groups/views.py

```diff
@@ -1,3 +1,161 @@
-from django.shortcuts import render
+from itertools import chain
 
-# Create your views here.
+from django.contrib.auth.mixins import PermissionRequiredMixin
+from django.contrib.messages.views import SuccessMessageMixin
+from django.contrib.auth import get_user_model
+from django.db.models import Count
+from django.shortcuts import get_object_or_404
+from django.urls import reverse_lazy, reverse
+from django.views import generic
+
+from groups.forms import GroupCreateForm
+from groups.models import Group
+from CultureAnalyzer.mixins import SafePaginationMixin
+
+PAGINATOR = 50
```

Comment:
can you change name of this variable?

Reply:
Changed to constants/ITEMS_ON_PAGE (imported from CultureAnalyser/constants)